envs/development/conda/fedbiomed-node.yaml

```diff
@@ -10,56 +10,55 @@ channels:
 
 dependencies:
   # minimal environment
-  - python >=3.9,<3.10
-  - pip
-  - jupyter
-  - ipython
+  # python 3.11 recently released 2022-11 and not yet supported by some deps including: torchvision
+  - python >=3.10,<3.11
+  - pip >= 23.0
+  - jupyter ~=1.0.0
```

Comment:
Hi @mvesin,

I know that it was already there but do we need jupyter installed on the node side?

Reply:
Thanks for noticing ... it is probably not needed. I remove it in all 3 node environments (dev/linux, dev/macos, vpn/linux)